### ISSUE TITLE
Closing private window in Firefox causes stream to stop

### ISSUE THREAD

**Issue by MeFri** [ampache/ampache#1120]
So, probably a Firefox issue, but anyways, maybe someone can point me in the right direction:
Everytime I close a private window in Firefox the current (audio) stream will stop shortly (<1s) after. Has anybody had similar issues? Happens on both Windows and Linux.
Ampache 3.8.2, Mozilla/5.0 (X11; Linux x86_64; rv:44.0) Gecko/20100101 Firefox/44.0

**Comment by Afterster** [comment]
Cannot reproduce, tried with flash/html5/javascript player tech but everything is working fine. I will let your issue open for few more days just in case but then I'm going to close it. Good luck!

**Comment by MeFri** [comment]
So I did a little more testing. Apparently the format has something to do with it. I used downsampling with opus encoding because of improved quality/bandwidth and all major browsers support it natively. When using mp3 instead the webplayer will still stutter, but continue playing, when I close a private window. I could not yet test with flash, but it happens with the html5 regardless of the aurora.js setting.
